File: dotnet/src/Azure/Mgmt/Microsoft.Compute/2016_01_31/Operations/VirtualMachinesOperations.cs
using Azure.Mgmt.Models;
using Azure.Mgmt.Compute._2016_01_31.Models;
using System.Collections.Generic;

namespace Azure.Mgmt.Compute._2016_01_31.Operations {
    public class VirtualMachinesOperations {
        public VirtualMachinesOperations(AzureClient client){

        }

        public IVirtualMachine Create(string name, string location, IDictionary<string, string> tags = null, string licenseType = null, IPlan plan=null){
            return new VirtualMachine(name, location, tags, licenseType, plan);
        }
    }
}

File: ruby/src/azure/mgmt/microsoft_compute/2016_01_31/virtual_machines/operations/virtual_machines_operations.rb
require_relative "../models/virtual_machine"

module Azure
  module Mgmt
    module Compute
      module Api_2016_01_31
        module Operations
          class VirtualMachinesOperations
            def initialize(client)
            end

            def create
              Azure::Mgmt::Compute::Api_2016_01_31::Models::VirtualMachine.new
            end
          end
        end
      end
    end
  end
end